apix-interface/src/main/java/eu/xenit/apix/comments/Comment.java

```diff
@@ -0,0 +1,109 @@
+package eu.xenit.apix.comments;
+
+import eu.xenit.apix.data.NodeRef;
+import java.util.Objects;
+
+public class Comment {
+
+    private NodeRef id;
+    private String title;
+    private String content;
+    private String createdAt;
+    private String createdBy;
+    private String modifiedAt;
+    private String modifiedBy;
+    private boolean canEdit;
+    private boolean canDelete;
+
+    public NodeRef getId() {
+        return id;
+    }
+
+    public void setId(NodeRef id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public String getContent() {
+        return content;
+    }
+
+    public void setContent(String content) {
+        this.content = content;
+    }
+
+    public String getCreatedAt() {
+        return createdAt;
+    }
+
+    public void setCreatedAt(String createdAt) {
+        this.createdAt = createdAt;
+    }
+
+    public String getCreatedBy() {
+        return createdBy;
+    }
+
+    public void setCreatedBy(String createdBy) {
+        this.createdBy = createdBy;
+    }
+
+    public String getModifiedAt() {
+        return modifiedAt;
+    }
+
+    public void setModifiedAt(String modifiedAt) {
+        this.modifiedAt = modifiedAt;
+    }
+
+    public String getModifiedBy() {
+        return modifiedBy;
+    }
+
+    public void setModifiedBy(String modifiedBy) {
+        this.modifiedBy = modifiedBy;
+    }
+
+    public boolean isCanEdit() {
+        return canEdit;
+    }
+
+    public void setCanEdit(boolean canEdit) {
+        this.canEdit = canEdit;
+    }
+
+    public boolean isCanDelete() {
+        return canDelete;
+    }
+
+    public void setCanDelete(boolean canDelete) {
+        this.canDelete = canDelete;
+    }
```

Comment:
The names of these methods are kind of strange. I know that `isSomething` is a convention for boolean getters, but `isCanDelete` and `setCanDelete` sound like you really want to say `isDeletable`/`setDeletable`.

Reply:
they were autogenerated, but I suppose I can clean it up

Reply:
I'm not sure how jackson would like those other names when serializing to json.

Reply:
I can refactor the property names to `deletable` . that should clean up somewhat and still work.